src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -107,7 +107,7 @@ define(["globalize", "dom", "emby-checkbox", "emby-select", "emby-input"], funct
         if (!plugins.length) return html;
 
         html += '<div class="metadataFetcher" data-type="' + availableTypeOptions.Type + '">';
-        html += '<h3 class="checkboxListLabel">' + globalize.translate("LabelTypeMetadataDownloaders", availableTypeOptions.Type) + "</h3>";
+        html += '<h3 class="checkboxListLabel">' + globalize.translate("LabelTypeMetadataDownloaders", globalize.translate(availableTypeOptions.Type)) + "</h3>";
```

Comment:
This does not look right...

Reply:
Okay nevermind I think this is fine, but it looks really odd to translate the fallback value also... do you also need a fallback for the translation of the fallback? :joy:

Reply:
You'd think that, but the type is also a translatable string, which was causing the issue for music libraries. Shows and movies were only one word so they never looked odd but music types were always untranslated.